commit f5ed28f536541d835df6ea8e014249199838ba17
Author: Jonathan Ellis <jbellis@apache.org>
Date:   Wed Sep 8 14:34:22 2010 +0000

    making endpoint cache volatile is unnecessary since we're using NBHM.  patch by jbellis
    
    git-svn-id: https://svn.apache.org/repos/asf/cassandra/trunk@995068 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/src/java/org/apache/cassandra/locator/AbstractEndpointSnitch.java b/src/java/org/apache/cassandra/locator/AbstractEndpointSnitch.java
index 8967fbb..c568076 100644
--- a/src/java/org/apache/cassandra/locator/AbstractEndpointSnitch.java
+++ b/src/java/org/apache/cassandra/locator/AbstractEndpointSnitch.java
@@ -38,7 +38,7 @@ public abstract class AbstractEndpointSnitch implements IEndpointSnitch
     /* list of subscribers that are notified when cached values from this snitch are invalidated */
     protected List<AbstractReplicationStrategy> subscribers = new CopyOnWriteArrayList<AbstractReplicationStrategy>();
     
-    private volatile Map<Token, ArrayList<InetAddress>> cachedEndpoints = new NonBlockingHashMap<Token, ArrayList<InetAddress>>();
+    private final Map<Token, ArrayList<InetAddress>> cachedEndpoints = new NonBlockingHashMap<Token, ArrayList<InetAddress>>();
     
     public ArrayList<InetAddress> getCachedEndpoints(Token t)
     {
